regular-expressions/introduction/basic-syntax-ii/regex-ranges-ii.md

```diff
@@ -0,0 +1,58 @@
+---
+author: emmab
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+---
+# Ranges II
+
+---
+## Content
+
+As with most patterns in regex, we can write the opposite of what we want in a very simple way.
+
+Say we want all letters and numbers outside of the ranges `a-c` and `1-3` then we can use the hat/caret character, `^`, inside the square brackets to exclude these characters.
+
+```
+/[^a-c1-3]/
+
+'abc123' ❌
+'def5678' ✅
+```
```

Comment:
I like this even more, could we make all examples like this? it looks really good in the app

@emmbow